dbus/dbus_libgio_constants.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
```

Comment:
Aren't we missing here the flag?
```suggestion
//    limitations under the License.
// +build !nodbus,cgo
```

Reply:
No, the whole point of having the constants in an independent file is to compile them in, without dbus.
This is because the app module is importing and using them.